domains/authorization/documentation/client_credentials.rest

```diff
@@ -15,7 +15,7 @@
 
 ###
 # @name tokenResponse
-POST https://datahubeouenerginet.b2clogin.com/datahubeouenerginet.onmicrosoft.com/B2C_1A_ClientCredentials/oauth2/v2.0/token HTTP/1.1
+POST https://login.microsoftonline.com/d3803538-de83-47f3-bc72-54843a8592f2/oauth2/v2.0/token HTTP/1.1
```

Comment:
Nit: Could we parametize the guid in the middle of the uri, so it can be changed at the top of the script, or wont that be needed?

Reply:
Done